Doc/library/crypt.rst

```diff
@@ -111,7 +111,7 @@ The :mod:`crypt` module defines the following functions:
    .. index:: single: crypt(3)
 
    Since a few :manpage:`crypt(3)` extensions allow different values, with
-   different sizes in the *salt*, it is recommended to use  the full crypted
+   different sizes in the *salt*, it is recommended to use the full encrypted
```

Comment:
Usage of the term "crypted" is deliberate here. There's a code example below with a variable "cryptedpasswd". It's a rare usage but valid:

https://www.google.com/search?q=%22crypted+password%22

The intended meaning is that a "crypted password" is a password that was passed through `crypt()`. Thus, it is not retrievable again, unlike regularly "encrypted" passwords which can still be "decrypted". Moreover, for this particular example,the  "crypted password" also contains an embedded salt which is crucial in checking whether the password is valid. Consider:

```pycon
>>> cleartext = 'scott-tiger'
>>> salt = crypt.mksalt()
>>> cryptedpasswd = crypt.crypt(cleartext, salt)
>>> is_password_valid = crypt.crypt(cleartext, cryptedpasswd) == cryptedpasswd
>>> is_password_valid
True
>>> is_password_valid = crypt.crypt('tiger-scott', cryptedpasswd) == cryptedpasswd
>>> is_password_valid
False
```

```suggestion
   different sizes in the *salt*, it is recommended to use  the full crypted
```